src/controllers/tripsController.js

```diff
@@ -131,6 +125,21 @@ class tripsController {
       errorResponse(res, 500, error.message);
     }
   }
+
+  static async getTrips(req, res) {
+    const { userID } = req.user;
+    try {
+      const trips = await travelRequests.findAll({
+        where: { userID },
+        attributes: {
+          exclude: ['id', 'userID', 'passportNumber', 'gender', 'createdAt', 'updatedAt']
+        },
+      });
+      return successResponse(res, 200, req.i18n.__('retrievedSuccessfully'), trips);
```

Comment:
```suggestion
      return successResponse(res, 200, setLanguage(req.user.preferedLang).__('retrievedSuccessfully'), trips);
```
@blaise82  I think that you'll have to keep track of translation basing on the user's preference when my PR is probably merged.

Reply:
that haven't been merged
so I don't think it applies now

Reply:
@blaise82 For sure. You'll address these changes when this [PR](https://github.com/andela/octopus-bn-backend/pull/28) is merged.